src/Client/Modals/SelectiveTemplateFromDB.fs

```diff
@@ -0,0 +1,104 @@
+namespace Modals.Template
+
+open Feliz
+open Feliz.DaisyUI
+open Model
+open Messages
+open Shared
+open Shared.DTOs.SelectedColumnsModalDto
+
+open ARCtrl
+open JsonImport
+open Components
+
+open Modals
+open Modals.ModalElements
+
+type SelectiveTemplateFromDBModal =
+
+    static member private LogicContainer (children: ReactElement list) =
```

Comment:
This exists somewhere else, right? Please reuse existing components, and move them to be available. And do not move into such a specialized space like this.

Reply:
fixed
It was still available in the SidebarComponents but that is far below the modal level.
Moved it into the Modalelements for now.
Maybe we should create a collection for such unspecialized containers above these elements.

Reply:
this is mostly used by the sidebar elements, not modals. You can use the `GenericComponents.fs` at client root level. The components from that file will be moved into the components project soon.